src/PrestaShopBundle/Resources/views/Admin/Module/Includes/modal_confirm.html.twig

```diff
@@ -53,7 +53,11 @@
                     {{ "Your current settings will be saved, but the module will no longer be active."|trans({}, 'Admin.Modules.Notification') }}
                   {% endif %}
                   {% if module_action == 'uninstall' %}
-                    {{ "You are about to uninstall %moduleName% module."|trans({'%moduleName%': module.attributes.displayName}, 'Admin.Modules.Notification') }}
+                    {% if module.attributes.confirmUninstall %}
+                      {{ module.attributes.confirmUninstall }}
+                    {% else %}
+                      {{ "You are about to uninstall %moduleName% module."|trans({'%moduleName%': module.attributes.displayName}, 'Admin.Modules.Notification') }}
```

Comment:
@fatmaBouchekoua, can you please paste this code ?

```php
{{ "You are about to uninstall %moduleName% module."|trans({'%moduleName%': module.attributes.displayName}, 'Admin.Modules.Notification') }}
<br>
{{ module.attributes.confirmUninstall }}
```

You may tell me that in some case the value can be undefined. We can add this property in:
https://github.com/PrestaShop/PrestaShop/blob/develop/src/Adapter/Module/Module.php#L84
```php
    private $attributes_default = array(
        [...]
        'confirmUninstall' => '',
```